### PR TITLE
build: fix mismatched npm versions

### PR DESCRIPTION
**Pull request by snario**
Fixes mismatched npm versions via `syncpack`

https://github.com/JamieMason/syncpack#fix-mismatches

### CODE REVIEW

**Comment by gakonst** [comment]
There seems to be a type incompatibility between sentry types and pino-sentry I believe.

File: packages/batch-submitter/src/exec/run-batch-submitter.ts [hunk 1]
```diff
@@ -129,9 +129,9 @@ export const run = async () => {
 
   const clearPendingTxs = requiredEnvVars.CLEAR_PENDING_TXS === 'true'
 
-  const l1Provider = new JsonRpcProvider(requiredEnvVars.L1_NODE_WEB3_URL)
+  const l1Provider = new providers.JsonRpcProvider(requiredEnvVars.L1_NODE_WEB3_URL)
   const l2Provider = injectL2Context(
-    new JsonRpcProvider(requiredEnvVars.L2_NODE_WEB3_URL)
+    new providers.JsonRpcProvider(requiredEnvVars.L2_NODE_WEB3_URL)
```
Comment (annieke) [inline packages/batch-submitter/src/exec/run-batch-submitter.ts:134]: doesn't need to be in this PR but seems like we may be able to deprecate usage of `@ethersproject/providers` across packages

**Comment by gakonst** [comment]
@snario this PR may be worth doing incrementally, there's lots of dependency hell with incompatible versions here...

**Comment by snario** [comment]
Yeah, let's do that.

**Pull request closed without merging**